engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/EngineJobExecutionActor.scala

```diff
@@ -360,8 +365,21 @@ class EngineJobExecutionActor(replyTo: ActorRef,
   }
 
   private def handleCacheInvalidatedResponse(response: CallCacheInvalidatedResponse, data: ResponsePendingData) = {
+    def updateMetadataForInvalidatedEntry(entry: CallCachingEntry) = {
+      import cromwell.services.metadata.MetadataService.implicits.MetadataAutoPutter
+      import cromwell.core.ExecutionIndex._
+      
+      val workflowId = WorkflowId.fromString(entry.workflowExecutionUuid)
+      // If the entry doesn't have an attempt, it means that this cache entry was added before this change
+      // and we don't know which attempt yielded this cache entry
+      // In that case make a best effort and update the first attempt
```

Comment:
ToL: I would say a best effort would be to update the last attempt (the one most likely to have actually succeeded)

OTOH, I imagine that's probably horrible... can we update *all* attempts?